qa/tasks/cephfs/mount.py

```diff
@@ -785,6 +785,10 @@ def run_shell(self, args, timeout=300, **kwargs):
                                       stdout=stdout, stderr=stderr,
                                       omit_sudo=omit_sudo, **kwargs)
 
+    def get_shell_stdout(self, args, timeout=300, **kwargs):
```

Comment:
Reduces typing effort, makes it easy to read. We have similar helper methods to run Ceph commands but not for shell commands on CephFS root.

Reply:
Let's drop this from this batch.

Reply:
This method is very useful in commits where tests have been added.

It reduces typing effort, makes it easy to read. We have similar helper methods to run Ceph commands but not for shell commands on CephFS mount.

Reply:
Let's have it as a separate change please?

Reply:
Updating this threads with summary of conversation on Slack, it is fine to add this method if/since this doesn't introduce any refactoring of the code.